src/app/pages/questions/services/audio-record.service.ts

```diff
@@ -1,17 +1,13 @@
 import { Injectable } from '@angular/core'
 import { File } from '@ionic-native/file/ngx'
-// NOTE: File path to opensmile.js; Adding opensmile plugin
-import * as opensmile from 'cordova-plugin-opensmile/www/opensmile'
 
-declare var cordova: any
-declare var window: any
+declare var Media: any // stops errors w/ cordova-plugin-media-with-compression types
+
 @Injectable()
 export class AudioRecordService {
-  filename: string = 'audio-opensmile.bin'
-
   audioRecordStatus: boolean = false
```

Comment:
if it is a boolean, I would called it `isRecordingAudio` or similar and the get method as `isRecording`